project-reports/2024/2024-Q3-Hyperledger-Bevel.md

```diff
@@ -0,0 +1,82 @@
+
+
+1. Getting code contributions for Hyperledger Bevel still remains a challenge.
+2. The Bevel project has received no external contributions; nearly all contributions for the last couple of releases have come from the Accenture team. Accenture will not be able to maintain the project without other organization or individual contributor help in maintaining.
+3. Community usage and interest are positive, as inferred from discussions and queries on the Discord channel.
+
+[Hyperledger Bevel Operator Fabric LFX Insights page](https://insights.lfx.linuxfoundation.org/foundation/hyp/overview?project=bevel&repository=https:%2F%2Fgithub.com%2Fhyperledger%2Fbevel-operator-fabric)
+
+
+# Questions/Issues for the TOC
+
+None.
+
+# Releases
+
+[Releases](https://github.com/hyperledger/bevel/releases) past quarter:
+
+- Bevel Release v1.0.1 and v1.1.0
+
+
+# Overall Activity in the Past Quarter
+
+The project delivered its roadmap targets for Q2/Q3. See [roadmap](https://hyperledger-bevel.readthedocs.io/en/develop/references/roadmap/) for details.
+
+Deployment using just helm commands with proxy/vault and without proxy/vault options for all DLT platform
+Ansible automations to use the new helm charts.
+
+# Current Plans
+
+- Continue with Q3 plan and release commitments.
+- Hyperledger meetup: Bevel workshop series remaining two sessions in September.
+- Explore cloud native vaults (AWS secret manager).
+- Collaborate with hyperledger-labs blockchain explorer and fabric-opertaions-console teams and plan to add deployment support on bevel.
+
+# Maintainer Diversity
+
+-	There is change in the maintainers for Bevel. You can find our current maintainer list here:  
+<a href="https://github.com/hyperledger/bevel/blob/main/MAINTAINERS.md" class="external-link" rel="nofollow">https://github.com/hyperledger/bevel/blob/main/MAINTAINERS.md</a>.
+
+<!-- -->
+
+-	Existing maintainers:
+	-	Sownak Roy (Accenture)
+	-	Suvajit Sarkar (Accenture)
+	-	Aditya Joshi (Walmart)
+-	Emeritus Maintainers:
+	-	Jonathan M Hamilton (Accenture)
+	-	Tracy Kuhrt (Accenture)
+	-	Jagpreet Singh Sasan (Accenture)
+	-	Arun S M (Walmart)
+	-	Weihong Ou (Walmart)
+
+-	There is no change in the maintainers Bevel-Operator-Fabric. You can find our current maintainer list here:  
+<a href="https://github.com/hyperledger/bevel-operator-fabric/blob/main/MAINTAINERS.md" class="external-link" rel="nofollow">https://github.com/hyperledger/bevel-operator-fabric/blob/main/MAINTAINERS.md</a>.
+
+<!-- -->
+
+-	Existing maintainers:
+	-	David Viejo (Kung Fu Software)
+	-	Aditya Joshi (Walmart)
+
+# Contributor Diversity
+[Dashboard]
+(https://insights.lfx.linuxfoundation.org/foundation/hyp/reports/contributors?project=bevel&repository=https:%2F%2Fgithub.com%2Fhyperledger%2Fbevel&routedFrom=Github)
```

Comment:
This markdown is not formatting correctly as a link, probably because of the extra return.

Reply:
Have fixed this